### PR TITLE
feat : button,input,image components

### PR DESCRIPTION
**Pull request by dipteekhd**
Review : https://githubbox.com/dipteekhd/KHD-UI/tree/button-input-image

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
good to merge.
suggestion : try using rems if you're planning for responsiveness

File: components/button/button.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="button.css">
+    <title>Document</title>
```
Comment (kart-c) [inline components/button/button.html]: I would suggest changing the title here

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        <div class="radio-options radio-options--col m-sm-vr">
+            <label for="id11" class="m-sm-t">
+                <span><input id="id11" type="radio" name="rank" checked></span>
+                <span>First</span>
+            </label>
+
+            <label for="id12" class="m-sm-t">
+                <span><input id="12" type="radio" name="rank"></span>
+                <span>Second</span>
```
Comment (kart-c) [inline components/input/input.html]: give unique `id` attribute here as clicking on the 3/ `label` selects the 1/st `input`. Also the 2nd and 4th `label-for` attribute and `iput-id` is not the same so they are not working

**Review by kart-c — CHANGES_REQUESTED** [review]

File: components/input/input.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        </div>
+
+        <div class="radio-options radio-options--col m-sm-vr">
+            <label for="id13" class="m-sm-t">
+                <span><input id="id13" type="radio" name="rank" checked></span>
+                <span>First</span>
+            </label>
+
+            <label for="id14" class="m-sm-t">
+                <span><input id="14" type="radio" name="rank"></span>
```
Comment (kart-c) [inline components/input/input.html]: `id` and `for` attribute are not same

**Review by kart-c — APPROVED** [review]
Please check the `id` and `for` attributes or this will cause problems to the user of your CL

**Pull request merged**